scripts/build/termux_step_get_dependencies.sh

```diff
@@ -21,11 +23,9 @@ termux_step_get_dependencies() {
 			# llvm doesn't build if ndk-sysroot is installed:
 			if [ "$PKG" = "ndk-sysroot" ]; then continue; fi
 			read DEP_ARCH DEP_VERSION DEP_VERSION_PAC <<< $(termux_extract_dep_info $PKG "${PKG_DIR}")
-
 			if [ ! "$TERMUX_QUIET_BUILD" = true ]; then
 				echo "Downloading dependency $PKG@$DEP_VERSION if necessary..."
 			fi
-
 			if [ -e "$TERMUX_BUILT_PACKAGES_DIRECTORY/$PKG" ]; then
```

Comment:
You need to remove this `if`, packages will never be  force built this way. I mentioned it in https://github.com/termux/termux-packages/pull/13861#issuecomment-1350941945

Reply:
Thank